src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.OpenSsl.cs

```diff
@@ -288,6 +292,14 @@ internal static bool DoSslHandshake(SafeSslHandle context, byte[] recvBuf, int r
                 {
                     sendCount = BioRead(context.OutputBio, sendBuf, sendCount);
                 }
+                catch (Exception e)
```

Comment:
should BioRead even run when exception occurred?

Reply:
I don't understand your question. We need BioRead() to get alert if handshake fails and openssl decides to send one.

Reply:
right, I missed the comment above